src/DependencyRewriterPlugin.php

```diff
@@ -71,6 +70,16 @@ public function activate(Composer $composer, IOInterface $io)
         $this->output(sprintf('<info>Activating %s</info>', __CLASS__), IOInterface::DEBUG);
     }
 
+    public function deactivate(Composer $composer, IOInterface $io) : void
```

Comment:
we can't have `: void` as we still on PH 5.6 and Composer v2 is gonna still support PHP 5.X

Reply:
`: void` removed.